examples/secp256k1-usdt/src/index.tsx

```diff
@@ -0,0 +1,148 @@
+import "bulma/css/bulma.css";
+
+import React, { FC, ReactNode, useEffect, useMemo, useState } from "react";
+import ReactDOM from "react-dom";
+import {
+  fetchCKBBalance,
+  fetchSUDTBalance,
+  generateAccountFromPrivateKey,
+  issueSUDT,
+  transferCKB2SUDT,
+  CKB2SUDTRate,
+} from "./lib";
+
+const Field: FC<{ label: string; value: string; onChange: React.ChangeEventHandler<HTMLInputElement> }> = ({
+  label,
+  value,
+  onChange,
+}) => {
+  return (
+    <div className="field">
+      <label className="label">{label}</label>
+      <div className="control">
+        <input className="input" type="text" value={value} onChange={onChange} />
+      </div>
+    </div>
+  );
+};
+const Notification: FC<{ onClose: () => void; children: ReactNode }> = ({ children, onClose }) => {
+  return (
+    <div className="notification is-success">
+      <div className="delete" onClick={onClose}></div>
+      {children}
+    </div>
+  );
+};
+
+const App: FC = () => {
+  const [issuerPrivateKey, setIssuerPrivateKey] = useState(
+    "0x5a85f2c71da9ca9581dfdf59c19dba77005b9d9a61b3c647a6a122bee1be409e"
+  );
+  const [holderPrivateKey, setHolderPrivateKey] = useState(
+    "0xf35458a0da78428e93858c996097afed03d61b9b3504ed56be7202ed83f45260"
+  );
```

Comment:
```suggestion
  const [issuerPrivateKey, setIssuerPrivateKey] = useState("");
  const [holderPrivateKey, setHolderPrivateKey] = useState("");
```

a hard-code and plaintext private key is not a good practice